src/audio/component.c

```diff
@@ -239,3 +239,36 @@ void sys_comp_init(void)
 		((void(*)(void))(*comp_init))();
 	}
 }
+
+int comp_copy_helper(struct comp_dev *dev, struct comp_copy_util *u)
```

Comment:
@singalsu this could probably be a separate patch and applied to other comps that do the same thing (volume, selector etc)

Reply:
ok!

Reply:
naming here could be improved tp be more meaningful. comp_get_copy_limits()

Reply:
ok!